src/ray/common/ray_config_def.h

```diff
@@ -80,7 +80,7 @@ RAY_CONFIG(uint64_t, raylet_check_gc_period_milliseconds, 100)
 /// memory_usage_threshold_fraction and free space is below the min_memory_free_bytes then
 /// it will start killing processes to free up the space.
 /// Ranging from [0, 1]
-RAY_CONFIG(float, memory_usage_threshold_fraction, 0.9)
+RAY_CONFIG(float, memory_usage_threshold_fraction, 0.98)
```

Comment:
isn't it too high? I'd imagine OS OOM killer can kick in before our OOM killer does something what about 0.95 (which is the same as existing oom killer)?

Reply:
I think this minimizes regression where we trigger the killing too early and is causing workloads to regress. If this is run in a container with memory limit it is ok if the usage goes above the total as that is just a soft limit